roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DiskAvailability(OpenShiftCheck):
+    """Check that recommended disk space is available."""
```

Comment:
Not sure how else we should annotate it but it should be clear that this is disk space prior to an install. If running preflight checks before an upgrade, they may have used a bunch of the space and that should be fine; we should probably not even run this check in that case. We would do a different kind of calculation for hosts that are already installed and running.